src/server/services/status.js

```diff
@@ -11,13 +11,25 @@ export async function getAllStatus() {
 }
 
 /**
+ * Fetch a service by its name.
+ *
+ * @param  {String} id
+ * @return {Promise}
+ */
+export async function getStatusGroupedByName() {
+  return await new Status().query(qb => qb.groupBy('name')
```

Comment:
Okay, I was just referring this for consistency. This is a simple query it's not a problem here, but for complex queries it wouldn't be possible with just query builder, in addition it would make the code messy.  

@deepsadhi So, for this one you may keep it as it is. But remove the unnecessary `async` and `await` in this function as this function already returns a promise.